<?xml version="1.0"?>
<launch>
  <!-- turtlebot_teleop_key already has its own built in velocity smoother -->
<!-- <node name="slam_gmapping" pkg="gmapping" type="slam_gmapping">
     <param name="scan_height" value="10"/> 
     <param name="scan_time" value="0.033"/>
     <param name="range_min" value="0.45"/>
     <param name="range_max" value="10.0"/>
     <param name="min_height" value=".10"/>
     <param name="max_height" value="2.0"/>
     <param name="output_frame_id" value="/base_link"/>
     <remap from="/scan" to="/scan"/> 
     <param name="map_frame" value="map" />
     <param name="_odom_frame" value="odom" />
     <param name="_base_frame" value="/base_link" />
     <param name="_inverted_laser" value="false" />
  </node> -->
<!-- Run the Transforms --> 
<!--<node pkg="tf" type="static_transform_publisher" name="base_link_to_laser" args="0.23 0 0.17 0 0 0 base_link laser 100" />
<node pkg="tf" type="static_transform_publisher" name="link1_broadcaster" args="0.15 0 0.44 0 0 0 /zed_initial_frame /base_link  500" /> 
-->
  <node pkg="turtlebot_teleop" type="turtlebot_teleop_key" name="turtlebot_teleop_keyboard"  output="screen">
    <param name="scale_linear" value="0.5" type="double"/>
    <param name="scale_angular" value="1.5" type="double"/>
    <remap from="turtlebot_teleop_keyboard/cmd_vel" to="RosAria/cmd_vel"/>
  </node>
</launch>
